src/include/sof/spinlock.h

```diff
@@ -68,25 +69,20 @@
 #include <sof/trace/trace.h>
 #include <ipc/trace.h>
 #include <user/trace.h>
-#include <stdint.h>
 
 #define DBG_LOCK_USERS		8
 #define DBG_LOCK_TRIES		10000
 
-#define trace_lock(__e)		trace_error_atomic(TRACE_CLASS_LOCK, __e)
-#define tracev_lock(__e)	tracev_event_atomic(TRACE_CLASS_LOCK, __e)
-#define trace_lock_error(__e)	trace_error_atomic(TRACE_CLASS_LOCK, __e)
+#define trace_lock(__e, ...) \
```

Comment:
Please add these to a separate commit to be easier to understand.

Reply:
I hope I didn't mess up anything because this split was actually nontrivial.